sim/src/lib.rs

```diff
@@ -142,6 +142,14 @@ impl AgentID {
             AgentID::BusPassenger(_, _) => AgentType::TransitRider,
         }
     }
+
+    pub fn to_vehicle_type(self) -> Option<VehicleType> {
+        match self {
+            AgentID::Car(c) => Some(c.1),
+            AgentID::Pedestrian(_) => None,
+            AgentID::BusPassenger(_, _) => None,
```

Comment:
Does this mean that a BusPassenger and a Pedestrian are rendered the same while zoomed out? vs. a car which is rendered differently?